src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt

```diff
@@ -33,34 +33,21 @@ class EvolvePokemon : Task {
 
                 if (pokemonMeta.candyToEvolve > 0) {
                     maxPossibleEvolves = bot.api.inventories.candyjar.getCandies(pokemonMeta.family) / pokemonMeta.candyToEvolve
-                }
-                
-                // Add the minimum value, depending on which is the bottleneck, amount of candy, or pokemon of this type in pokebank:
-                countEvolveStack +=  Math.min(maxPossibleEvolves,it.value.count())
-
-                // Use Iv sorting if this is configured:
-                val sorted = if (settings.sortByIv) {
-                    it.value.sortedByDescending { it.getIv() }
                 } else {
-                    it.value.sortedByDescending { it.cp }
-                }
-                // Release pokemon we wont be able to evolve anyway, because of lack of candy:
-                sorted.forEach {
-                    maxPossibleEvolves--
-                    if (maxPossibleEvolves < 0) {
-                        releasePokemon(it,"Not enough candy to save for evolve",ctx)
-                    }
+                    Log.red("${it.key} is in evolve list but is unevolvable")
```

Comment:
it's what i want... means you are stupid if you put a max evolved pokemon in the list